### DISCUSSION TITLE
higress 按照官网提示无法正确部署

### DISCUSSION THREAD

**Discussion by dnmyGo** [higress-group/higress#311] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37365340/236592239-6e62d1c2-c527-4776-a3c8-bf6b7bd318e7.png)
安装方式如上图，按图中给出的命令依次执行
当执行kubectl --namespace higress-system port-forward $POD_NAME 8080:$CONTAINER_PORT 会阻塞如下图。
![image](https://user-images.githubusercontent.com/37365340/236592363-d1ef25f2-047c-4114-af53-7f4f11ba7957.png)

输入http://127.0.0.1:8080  无法访问
2：查看service情况如下图
![image](https://user-images.githubusercontent.com/37365340/236592576-a6d0d66d-34b5-45ab-910c-a23341e83662.png)

描述该pods 如下图
![image](https://user-images.githubusercontent.com/37365340/236592608-4115ecac-a428-45c7-86a9-ecfdcb6219f1.png)
![image](https://user-images.githubusercontent.com/37365340/236592619-8db99b75-60e2-4824-9c76-ba2c531a714b.png)

**Comment by CH3CHO** [comment, marked as answer]
如果你用的是本地模式进行的安装，那么是不需要 port-forward 的，本身他就会监听 80 端口的。

能看一下 `kubectl get po -n higress-system` 的输出吗？ controller 是不是都已经启动起来了？个人觉得这个 Warning 可能不是问题，毕竟后面已经 Normal 了。

**Reply by dnmyGo** [reply]
![image](https://user-images.githubusercontent.com/37365340/236593121-2ecdf45e-deb2-49ef-b119-d10780e1a3e8.png)

**Reply by CH3CHO** [reply]
这样的话应该没什么问题的。你可以直接请求 http://localhost/ 试试。如果返回 404 就说明 OK 了。

**Reply by CH3CHO** [reply]
ClusterIP 显示为 pending 应该是由于本地集群不支持 LoadBalancer 所致，是正常情况。本地模式安装的时候已经配置了 NodePort， 所以没有 LB IP 也没事。

**Reply by dnmyGo** [reply]
![image](https://user-images.githubusercontent.com/37365340/236593546-e7fde214-c924-4995-aa23-620a035937d7.png)
80 端口好像没开放，我让运维看下，谢谢

**Reply by CH3CHO** [reply]
哦哦，看到了，你是用的标准模式安装。这样的话是不会有 NodePort的。可以执行下面的命令把端口转发出来：
```shell
kubectl port-forward service/higress-gateway -n higress-system 80:80 443:443
```

**Reply by dnmyGo** [reply]
ok

**Reply by dnmyGo** [reply]
谢谢啊，能访问了
